src/mpc-hc/PPageAdvanced.cpp

```diff
@@ -187,6 +187,7 @@ void CPPageAdvanced::InitSettings()
         std::make_pair(10, 30), StrRes(IDS_PPAGEADVANCED_SCORE));
     addIntItem(AUTO_DOWNLOAD_SCORE_SERIES, IDS_RS_AUTODOWNLOADSCORESERIES, 0x18, s.nAutoDownloadScoreSeries,
         std::make_pair(10, 30), StrRes(IDS_PPAGEADVANCED_SCORE));
+    addBoolItem(RELOAD_AFTER_LONG_PAUSE, IDS_RS_OPEN_REC_PANEL_WHEN_OPENING_DEVICE, true, s.bOpenRecPanelWhenOpeningDevice, StrRes(IDS_PPAGEADVANCED_OPEN_REC_PANEL_WHEN_OPENING_DEVICE));
```

Comment:
"RELOAD_AFTER_LONG_PAUSE" is wrong here. You need to add a new value to the enum.

Reply:
Oh, damn. I messed that one up. Fixing it in a moment...